src/DashboardService/Controllers/CommentController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using LT.DigitalOffice.DashboardService.Business.Comment;
+using LT.DigitalOffice.DashboardService.Models.Dto.Models;
+using LT.DigitalOffice.DashboardService.Models.Dto.Requests.Comment;
+using LT.DigitalOffice.DashboardService.Models.Dto.Responses;
+using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.DashboardService.Controllers;
+
+[Route("[controller]")]
+[ApiController]
+public class CommentController : ControllerBase
+{
+  [HttpPost("create")]
+  public async Task<OperationResultResponse<Guid?>> CreateAsync(
+    [FromBody] CreateCommentRequest request,
+    [FromServices] CreateCommentCommand command)
+  {
+    return await command.ExecuteAsync(request);
+  }
+
+  [HttpGet("get")]
+  public async Task<FindResultResponse<IEnumerable<CommentInfo>>> GetAsync(
+    [FromServices] GetAllCommentsCommand command)
+  {
+    return await command.ExecuteAsync();
+  }
+
+  [HttpGet("get/{id}")]
+  public async Task<OperationResultResponse<CommentResponse>> GetAsync(
+    [FromRoute] Guid id,
+    [FromServices] GetCommentCommand command)
```

Comment:
also use FindFilter - check here for example https://github.com/LT-Students/DashboardService/pull/11/files#diff-f43852d0b5f83ea8e56c9064135f3d63c7739ad73355517bd8db78ff02613606R7